wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession.swift

```diff
@@ -384,6 +396,27 @@ public final class ZMUserSession: NSObject {
         self.contextStorage = contextStorage
         self.recurringActionService = recurringActionService
         self.dependencies = dependencies
+
+        super.init()
+
+        setupAnalyticsSession()
+    }
+
+    private func setupAnalyticsSession() {
+        let analyticsUserProfile: AnalyticsUserProfile = AnalyticsUserProfile(
+            analyticsIdentifier: selfUser.remoteIdentifier.uuidString,
+            teamID: selfUser.membership?.team?.remoteIdentifier?.uuidString,
+            teamRole: TeamRole.stringValue(from: selfUser.teamRole.rawValue),
+            teamSize: selfUser.membership?.team?.members.count,
+            contactCount: selfUser.membership?.team?.members.count
```

Comment:
This is only needed if we actually create an analytics session, so I suggest to move this work inside the if.

Reply:
Done: https://github.com/wireapp/wire-ios/commit/7caa26a9f21c0b50fca5b69418abbd477f28e8ac